prow/config/config.go

```diff
@@ -441,6 +449,21 @@ type Controller struct {
 	AllowCancellations *bool `json:"allow_cancellations,omitempty"`
 }
 
+// GetDefaultReportTemplate returns the template that belong to a specific repository.
+// If the repository doesn't exist in the report_templates configuration it will
+// inherit the values from its organization, otherwise the default values will be used.
+func (c *Controller) GetDefaultReportTemplate(org, repo string) *template.Template {
```

Comment:
Can we rename this to `ReportTempateForRepo`? This isn't necessarily a default

Reply:
it is the default since we pass the org, repo vars. Also if there isn't any it will return the '*' default values which they are not related to any repo.

Reply:
By your definition, we only have defaults. As mentioned, this isn't necessarily a default, because we may get something that is more specific than `'*'`